Philips.CodeAnalysis.Common/AdditionalFilesHelper.cs

```diff
@@ -85,14 +85,12 @@ private string GetRawValue(string settingKey)
 			SyntaxTree tree = _compilation.SyntaxTrees.First();
 			AnalyzerConfigOptions analyzerConfigOptions = _options.AnalyzerConfigOptionsProvider.GetOptions(tree);
 
-#nullable enable
 			if (analyzerConfigOptions.TryGetValue(settingKey, out var value))
```

Comment:
Is there a test for this change? (ie retrieving an option that does not exist)

Reply:
Added a specific test case for this.